integration/spark2/src/main/scala/org/apache/spark/sql/CarbonEnv.scala

```diff
@@ -125,6 +127,19 @@ class CarbonEnv {
     LOGGER.info("Initialize CarbonEnv completed...")
   }
 
+
+  def createIndexServerTmp(sparkSession: SparkSession): Unit = {
+    val path = CarbonUtil.checkAndAppendFileSystemURIScheme(
+      "/tmp/" + CarbonCommonConstants.INDEX_SERVER_TEMP_FOLDER_NAME)
+    if (!FileFactory.isFileExist(path)) {
+        // Create the new directory
+        LOGGER.info(s"Creating Index Server Tmp folder: ${path} ")
```

Comment:
either give `tmp` or `temp` in log, remove Tmp

Reply:
done